components/src/NavBar/MobileMenu.js

```diff
@@ -12,42 +12,128 @@ import MenuLink from "./MenuLink";
 import theme from "../theme";
 import { subPx } from "../Utils";
 
-const SubMenuItemsList = styled.ul(({ isTopLayer }) => ({
+const ApplyMenuLinkStyles = styled.li({
+  "& *": {
+    display: "block",
+    color: theme.colors.white,
+    fontSize: theme.fontSizes.large,
+    lineHeight: theme.lineHeights.sectionTitle,
+    width: "100%",
+    justifyContent: "flex-start",
+    padding: `${theme.space.x2} ${theme.space.x3} ${theme.space.x2} ${theme.space.x3}`,
+    borderRadius: "0",
+    textDecoration: "none",
+    "&:hover, &:focus": {
+      outline: "none",
+      backgroundColor: theme.colors.black,
+    },
+    "&:disabled": {
+      opacity: ".5",
+    },
+  },
+});
+
+const MobileMenuLink = styled(MenuLink)({
+  fontSize: theme.fontSizes.large,
+  lineHeight: theme.lineHeights.sectionTitle,
+  width: "100%",
+  justifyContent: "flex-start",
+  padding: `${theme.space.x2} ${theme.space.x3} ${theme.space.x2} ${theme.space.x3}`,
+  borderRadius: "0",
+});
+
+const ApplySubMenuLinkStyles = styled.li(({ layer }) => ({
+  color: theme.colors.black,
+  justifyContent: "center",
+  alignItems: "center",
+  textDecoration: "none",
+  lineHeight: theme.lineHeights.base,
+  transition: ".2s",
+  fontSize: theme.fontSizes.medium,
+  "& *": {
+    display: "block",
+    color: theme.colors.white,
+    textDecoration: "none",
+    padding: `${theme.space.x1} ${theme.space.x2}`,
+    paddingLeft: `${(24 * layer) + 24}px`,
```

Comment:
I would extract the padding logic into a function. I see it appear several times.